reportportal/values.yaml

```diff
@@ -249,17 +302,21 @@ postgresql:
     port: 5432
     user: rpuser
     dbName: reportportal
-    ssl: disabled
     ## Whether or not to use SSL
+    ssl: disabled
     # ssl: disabled / require
     password:
     ## Number of database connections
     connections: 
 
 elasticsearch:
+  secretName: ""
   installdep:
     enable: false
-  endpoint: http://elasticsearch-master:9200
+  endpoint: elasticsearch-master.default.svc.cluster.local
+  port: 9200
+  user: elastic
+  password:
```

Comment:
Not good decision to make it like that. Many of our clients have Elastic under HTTPS, E.G. those who use managed AWS instances. We will break their deployments preventing them from upgrades.